template/app/src/server/scripts/dbSeeds.ts

```diff
@@ -0,0 +1,40 @@
+import { type User } from 'wasp/entities';
+import { faker } from '@faker-js/faker';
+import type { PrismaClient } from '@prisma/client';
+import { TierIds } from '../../shared/constants.js';
+
+/**
+ * This function, which we've imported in `app.db.seeds` in the `main.wasp` file,
+ * seeds the database with mock users via the `wasp db seed` command.
+ * For more info see: https://wasp-lang.dev/docs/data-model/backends#seeding-the-database
+ */
+export async function seedMockUsers(prismaClient: PrismaClient) {
+  await Promise.all(generateMockUsersData(50).map((user) => prismaClient.user.create({ data: user })));
```

Comment:
Nitpick: use `data` instead of `user` both to shorten it and to make it a tidbit more semantically correct.
Btw I am a bit divided on this one if it should be one line or not -> but at the end doesn't really matter much, so both is ok.

```suggestion
  await Promise.all(generateMockUsersData(50).map(
    data => prismaClient.user.create({ data })
  ));
```